src/main/resources/kibana-rest.yml

```diff
@@ -62,3 +62,43 @@ kibana_user:
   versions:
     ">= 7.6.0": "/internal/security/me"
     ">= 5.0.0 < 7.6.0": "/api/security/v1/me"
+
+kibana_security_exception_list:
+  versions:
+    ">= 7.10.0": "/api/exception_lists/_find?namespace_type=agnostic"
+
+kibana_security_endpoint_trusted_apps:
+  versions:
+    ">= 7.10.0": "/api/endpoint/trusted_apps"
+
+kibana_security_endpoint_host_isolation:
+  versions:
+    ">= 7.14.0": "/api/exception_lists/items/_find?list_id=endpoint_host_isolation_exceptions&namespace_type=agnostic"
+
+kibana_security_endpoint_event_filters:
+  versions:
+    ">= 7.14.0": "/api/exception_lists/items/_find?list_id=endpoint_event_filters&namespace_type=agnostic"
+
+kibana_security_endpoint_trusted_apps:
+  versions:
+    ">= 7.14.0": "/api/exception_lists/items/_find?list_id=endpoint_trusted_apps&namespace_type=agnostic"
+
+kibana_security_packages:
+  versions:
+    ">= 7.10.0": "/api/fleet/epm/packages?experimental=true&category=security"
+
+kibana_fleet_agents:
+  versions:
+    ">= 7.10.0": "/api/fleet/agents"
+
+kibana_fleet_agent_policies:
+  versions:
+    ">= 7.10.0": "/api/fleet/agent_policies"
+
+kibana_fleet_package_policies:
+  versions:
+    ">=7.10.0": "/api/fleet/package_policies"
```

Comment:
We've been sorting these keys alphabetically, so `kibana_f` should be sorted up above.